[Feat] 문서 작성화면 UI 구현

bynmch opened the pull request:
<!--
[타입] 간결하고 명확한 제목 (최대 70자)

예시:
[Feat] 카카오맵 기반 여행지 검색 및 추가 기능 구현
[Fix] 로그인 후 리다이렉션 오류 수정
[Refactor] FinalDraw 컴포넌트 composable 패턴으로 리팩토링
-->

## 📋 작업 내용
<!--
> 이 PR에서 구현한 기능이나 수정한 내용을 간단히 요약해주세요.
-->
- 결재 문서 작성화면 UI 구현
- 결재선 자동지정 렌더링
- 도장 영역 양방향 바인딩
- 참조목록 자동지정 렌더링
- 모달창 UI HTML/CSS 만 구현

## 🔧 작업 상세
<!--
> 주요 변경 사항을 구체적으로 작성해주세요.
-->

### 변경된 파일
- `/src/views/approval/create/ApprovalCreate.vue` -
- `/src/views/approval/create/ApprovalCreateCommon.vue` -  컴포넌트가 마운트 될 때 작성 화면 조회 API를 호출하도록 연동했으며, 결재선, 참조목록이 자동으로 렌더링되도록 했고, `watch`로 화면의 데이터 변화를 감지하도록 하였습니다.
- `/src/assets/styles/approval/commonform.css` - `ApprovalCreateCommon.vue`의 도장 영역 및 참조의 스타일을 좀 더 자연스럽게 수정했습니다.
<!-- `src/components/FinalDraw.vue`
-->
  
  <!--`src/composables/useKakaoMap.js`
  -->


### 주요 로직 설명
- 마운트될 때 호출한 작성화면 API를 통해 받은 데이터를 렌더링하도록 했습니다.
```
1. ApprovalCreate.vue onMounted
   ↓
2. approvalStore.fetchTemplate(4)
   ↓
3. API: GET /api/approval/templates/4
   ↓
4. template.value 저장 (Pinia store)
   ↓
5. watch 감지!
   ↓
6. initializeData() 재실행
   ↓
7. commonData.lines 업데이트
   ↓
8. UI 렌더링 ✅
```

## ✅ 테스트 체크리스트
- [x] 로컬 환경에서 정상 동작 확인
- [x] 주요 기능 시나리오 테스트 완료
- [x] 브라우저 콘솔 에러 없음
- [x] 반응형 디자인 확인 (모바일/태블릿/데스크탑)
- [ ] 크로스 브라우저 테스트 (Chrome, Safari, Firefox)

## 🖼️ 스크린샷 (UI 변경 시)
<!--
> UI/UX 변경이 있는 경우 스크린샷이나 GIF를 첨부해주세요.
-->

### Before
<!-- 변경 전 이미지 -->
<img width="1083" height="720" alt="image" src="https://github.com/user-attachments/assets/ca87fa5d-72f9-4dd7-8da0-52034157ade8" />

### After
<!-- 변경 후 이미지 -->
<img width="1318" height="826" alt="image" src="https://github.com/user-attachments/assets/2eca1645-8e50-4e9e-9b2a-01672015c708" />

### Plus
- 아직 사원정보 목록을 조회하는 API가 개발 안 되어있어서 그것 까지 하면 이게 가능할 것 같습니다.
<img width="1903" height="867" alt="image" src="https://github.com/user-attachments/assets/f96558be-21dd-4a9e-8182-54e5bac49295" />


## 🤔 리뷰 포인트
<!--
리뷰어가 특별히 집중해서 봐줬으면 하는 부분을 적어주세요.
-->
- 코딩 컨벤션 봐주세요.

## 🔗 관련 이슈
<!--  
아래 형식으로 적으면, 이 PR이 develop에 머지될 때 이슈가 자동으로 닫힙니다.
>
-->
- Closes #40 
- Closes #41
- Closes #132 
- Related to #132


## 💬 추가 코멘트
<!--
> 리뷰어에게 전달하고 싶은 추가 내용이 있다면 자유롭게 작성해주세요.
-->
- 제가 개발이 좀 늦어지고 있는데 이번 주말까지는 모두 완성할 수 있도록 노력하겠습니다.

bynmch committed:
[Feat] 문서 작성화면 UI 구현

- 결재 문서 작성화면 UI 구현
- 결재선 자동지정 렌더링
- 도장 영역 양방향 바인딩
- 참조목록 자동지정 렌더링
- 모달창 UI HTML/CSS 만 구현

작성화면 제목/분류

모달창 html/css

결재문서작성화면UI진행중

- 결재선 추가
- 참조 추가
- 스크립트 추가 + 하드코딩

작성화면 결재선/참조목록 자동지정  렌더링

작성화면 마운트 시 결재선도 자동지정되어 렌더링
참조목록도 자동지정되어 렌더링
결재 관리 설정 시 이상 데이터 삽입되는 로직 수정

bynmch committed:
[Feat] 문서 작성화면 UI 구현

- 결재 문서 작성화면 UI 구현
- 결재선 자동지정 렌더링
- 도장 영역 양방향 바인딩
- 참조목록 자동지정 렌더링
- 모달창 UI HTML/CSS 만 구현

작성화면 제목/분류

모달창 html/css

결재문서작성화면UI진행중

- 결재선 추가
- 참조 추가
- 스크립트 추가 + 하드코딩

작성화면 결재선/참조목록 자동지정  렌더링

작성화면 마운트 시 결재선도 자동지정되어 렌더링
참조목록도 자동지정되어 렌더링
결재 관리 설정 시 이상 데이터 삽입되는 로직 수정

indy0322 reviewed (dismissed):
👍 [칭찬]
문서 작성 페이지 UI가 사용자가 사용하기 편하게 잘 만들어진 것 같습니다.
사원정보 목록을 조회하는 모달까지 구현되었을 때의 페이지의 UI 및 기능이 기대가 됩니다.

bynmch committed:
[Feat] 상신/임시저장 UI 구현 및 API 연동

전체 조회는 되고, 트리 클릭이벤트는 안됨

결재선/참조목록 지정을 위한 모달창 UI 완성

상신API 연동

임시저장 API연동

bynmch committed:
[Feat] 상신/임시저장 UI 구현 및 API 연동

전체 조회는 되고, 트리 클릭이벤트는 안됨

결재선/참조목록 지정을 위한 모달창 UI 완성

상신API 연동

임시저장 API연동

haenin committed:
[Fix] WebSocket JWT 토큰 전송 누락으로 세션 등록 불가 문제 해결 (#159)

* [Feat] 알림 설정 페이지 구현 및 백엔드 API 연동

- 알림 설정 API 함수 추가 (findSettings, modifySettings)
- NotificationSettingsDTO 타입 정의
- notificationSettingsStore 구현 (백엔드 연동)
- 브라우저 알림 권한 자동 동기화 로직 추가
- DB 설정 우선시 적용

* [Fix] Api 수정

* [Fix] Api 수정

* [Refactor] api 복수형으로 수정

* [Fix] 파일명 수정

* [Fix] authStore에서 토큰을 가져와 백엔드에게 넘겨줌

* [Fix] develop 최신화

* [Fix] 로그인 시작점부터 웹소켓 연결

* [Fix] false파라미터 불필요

Easy-going12 reviewed (dismissed):
👍 [칭찬]

const emit = defineEmits(['close']);
const handleClose = () => emit('close');

ApprovalLineModal.vue 코드에서모달을 스스로 닫는 게 아니라 “부모에게 닫아달라고 요청”하는 방식이라 상태 관리가 단순해지고 재사용성이 높아져 좋은 거 같습니다.

bynmch committed:
[Fix] rebase이전 push한 브랜치 병합
